Assets/HoloToolkit/Input/Scripts/Utilities/InteractionSourcePressInfo.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace HoloToolkit.Unity.InputModule
+{
+    public enum InteractionSourcePressInfo
```

Comment:
I'm not yet sure how I feel about recreating Unity's enum here. It'll add code maintenance whenever the platform changes, and it's a breaking change for anybody using the existing event data classes. What do we gain over wrapping all references with UNITY_WSA?

Reply:
> I'm not yet sure how I feel about recreating Unity's enum here.

It's not the first time [the Input System does this](https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/master/Assets/HoloToolkit/Input/Scripts/InputSources/SupportedInputInfo.cs).

> it's a breaking change for anybody using the existing event data classes.

Well that's why it's a dev branch right?

> What do we gain over wrapping all references with UNITY_WSA?

All previous projects on the master branch gaining forward compatibility, especially ones where people use the input system in cross platform apps.

Reply:
I don't believe SupportedInputInfo is a Unity enum.

Yeah, forward compatibility makes sense for cross platform, though all apps will still need to add a press type to their existing events.

>Well that's why it's a dev branch right?

Now we're talking! Haha 😜

Reply:
😂😂😂

> I don't believe SupportedInputInfo is a Unity enum.

Hmm, could be another I was thinking of?

> all apps will still need to add a press type to their existing events anyway.

Why? Isn't there a default type that automatically works?